src/layouts/NavBar.vue

```diff
@@ -1,87 +1,73 @@
-    <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
-      <v-img
-        alt="Vue logo"
-        src="@/assets/swap-logo.png"
-        width="90"
-        height="90"
-      />
-      <div class="w-100">
-        <div class="row w-100">
-          <div class="col-4 offset-8 d-flex justify-content-end">
-            <div
-              v-if="isConnected"
-              class="wallet-status navbar-item indicator badge-outline badge-pill"
-            >
-              <span aria-describedby="tooltip-status" class="fromNetwork">
-                {{ chainName }}
-              </span>
-            </div>
-            <div
-              v-if="isConnected"
-              class="wallet-status navbar-item badge-pill text-truncate"
-              style="width: 155px"
-            >
-              <span id="address">{{ this.account }}</span>
-            </div>
-          </div>
-          <div
-            v-if="!isConnected"
-            id="navbarResponsive"
-            class="navbar-collapse collapse"
+    <nav class="navbar navbar-dark bg-dark">
+      <div>
+        <v-img alt="RSK" src="@/assets/swap-logo.png" width="90" height="90" />
+      </div>
+      <div>
+        <div v-if="isConnected" class="menuItem badge-outline badge-pill">
+          <span aria-describedby="tooltip-status" class="fromNetwork">
+            {{ chainName }}
+          </span>
+        </div>
+        <div
+          v-if="isConnected"
+          class="menuItem badge-pill text-truncate"
+          style="width: 155px;"
+        >
+          <span id="address">{{ this.account }}</span>
+        </div>
+      </div>
+      <div v-if="!isConnected" class="menuItem">
+        <div class="menuItem ml-auto">
+          <button
+            id="logIn"
+            type="button"
+            class="btn btn-primary badge-pill"
+            @click="connectWalletClick"
           >
-            <div class="navbar-item ml-auto">
-              <button
-                id="logIn"
-                type="button"
-                class="btn btn-primary badge-pill"
-                @click="connectWalletClick"
-              >
-                Connect wallet
-              </button>
-            </div>
-          </div>
+            Connect wallet
+          </button>
         </div>
       </div>
     </nav>
   </header>
 </template>
 
 <script>
-import * as constants from "@/store/constants";
-import { createNamespacedHelpers } from "vuex";
+import * as constants from '@/store/constants'
+import { createNamespacedHelpers } from 'vuex'
 
-const { mapState, mapGetters, mapActions } = createNamespacedHelpers("session");
+const { mapState, mapGetters, mapActions } = createNamespacedHelpers('session')
 
 export default {
-  name: "NavBar",
+  name: 'NavBar',
   data() {
     return {
       sideNetworkConfig: null,
-    };
+    }
   },
   computed: {
-    ...mapState(["enabled"]),
-    ...mapState(["account"]),
-    ...mapState(["currentChain"]),
+    ...mapState(['enabled']),
```

Comment:
let's review your/our linting settings. I'm pretty sure you have different options than the rest of us.